src/content/UserPage/UserPage.js

```diff
@@ -69,9 +69,9 @@ const UserPage = () => {
                   <div className='video-container'>
                     <iframe src='https://player.vimeo.com/video/46398191'
                       title='How it Works'
-                      frameborder='0'
+                      frameBorder='0'
                       allow='autoplay; fullscreen'
-                      allowfullscreen />
+                      allowFullScreen />
```

Comment:
Those changes are not related to this issue, do you mind opening another issue & pull request for that?

Reply:
No problem. Should I delete the previous commit then?